doc/api/deprecations.md

```diff
@@ -95,12 +95,18 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
```

Comment:
If this word continues to appear in this file, it should have a content warning at the top, and/or a content warning and a click-through to view this section.